lang/syn/src/lib.rs

```diff
@@ -844,6 +855,11 @@ pub struct ConstraintMintDecimals {
     pub decimals: Expr,
 }
 
+#[derive(Debug, Clone)]
+pub struct ConstraintMintTokenProgram {
```

Comment:
Is there any advantage of defining a new constraint here and not use `ConstraintTokenProgram`?

Reply:
I thought I was following a convention - but on second look I think reusing the same struct is fine.